platform/security/sts/pom.xml

```diff
@@ -25,14 +25,8 @@
     <packaging>pom</packaging>
     <modules>
         <module>security-sts-realm</module>
-        <module>security-sts-crlinterceptor</module>
-        <module>security-sts-server</module>
-        <module>security-sts-samlvalidator</module>
-        <module>security-sts-x509validator</module>
-        <module>security-sts-usernametokenvalidator</module>
         <module>security-sts-ldapclaimshandler</module>
         <module>security-sts-ldaplogin</module>
-        <module>security-sts-x509delegationhandler</module>
         <module>security-sts-attributequeryclaimshandler</module>
         <module>security-sts-attributequerycommon</module>
         <module>security-sts-propertyclaimshandler</module>
```

Comment:
❗️ ❓ Isn't this PR supposed to be removing all STS related things?

Reply:
It is removing the STS server. I'm not renaming stuff in this PR. That will be in a separate PR.